### PR TITLE
Modification release

### PR DESCRIPTION
**Pull request by autra**
~~Still a WIP, don't merge yet :-)~~ I believe this first iteration is good enough.

Here's a set of modification for the website.

Still a lot to do, including:
- create several smaller banner images for smaller screen (and 3g connections).
- get rid of the annoying flickering effect at load (probably skel.js, do we really need this?)
- correct link in menus
- add link for blog
- add an article announcing v2.0 release
- replace the banner by an actual runnable itowns in the background (with just panoramic controls activated?). -> I've just tested, the result is fun, but not really usable yet.
- auto-generate the pages linking to all the examples.
- ~at least change the demo to showcase itowns2 and not 1.~

And I probably forgot a lot of stuff...

### CODE REVIEW

**Comment by autra** [comment]
@iTowns/reviewers this PR has already too many commits! Please r? 

I'll put the rest of my TODO in issues and will work through them one at a time.

File: index.html [hunk 1]
```diff
@@ -11,109 +41,36 @@
...
-        It includes :
-        <br />
-        <ul>
-            <li>250 Oriented Images (50 Panoramics, 192 MB)</li>
-            <li>1 patch (500 * 500m) of 3D textured city models (BATI3D, 50 MB)</li>
-            <li>Terrestrial PointCloud (20 Millions point, 400 MB)</li>
-            <li>JSON of building footprints and DTM (500 kB)</li>
+            <li>Connect to WMTS, WMS, TMS servers</li>
+            <li>Load 3dtiles, gpx, KML...</li>
+            <li>Coming soon: Point Clouds (from PotreeConverter or Lopocs), WFS, oriented images...</li>
```
Comment (peppsac) [inline index.html]: I'd remove the `(from PotreeConverter or Lopocs)`: no really useful

File: index.html [hunk 2]
```diff
@@ -11,109 +41,36 @@
...
-        <p><a href="https://vimeo.com/225236123">iTowns Release 2.0</a> from <a href="https://vimeo.com/user48543162">itowns</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
-    </section>
-
-    <section  id="demo" class="wrapper style2 container special-alt">
-        <header>
-            <h2>Try it !</h2>
-        </header>
-        <img id="thumb_demo" class="clickable_img" src="images/thumb_demo.png" alt="Demo itowns">
-        <iframe id="iframe_demo" src="" width="100%" height="400" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+        <div class="wrapper-responsive wrapper-responsive-16-9">
```
Comment (peppsac) [inline index.html]: I think I'd move the video at the top of the page, before the `What is it` section.

And I'd also remove this:
```
         <header>
             <h2>Watch <strong>video</strong> of iTowns.</h2>
         </header>
         <p>The following video shows iTowns in action.</p>
```

File: _includes/header.html [hunk 3]
```diff
@@ -6,14 +6,11 @@ <h1 id="logo"><a href="/">iTowns <span>by iTowns team</span></a></h1>
...
                 <ul>
-                    <li><a href="#what">What is it ?</a></li>
-                    <li><a href="#data_types">Supported data types</a></li>
-                    <li><a href="#features">Features</a></li>
-                    <li><a href="#sample_data">Sample data</a></li>
-                    <li><a href="#get_started">Getting started</a></li>
-                    <li><a href="#video">Video</a></li>
-                    <li><a href="#demo">Live demo</a></li>
-                    <li><a href="#credits">Support</a></li>
+                    <li><a href="/blog">Blog</a></li>
```
Comment (peppsac) [inline _includes/header.html]: Could you remove the `Blog` item?
(we can restore it if someone commits to write blog articles)

**Comment by autra** [comment]
@peppsac updated.

**Comment by peppsac** [comment]
Looks good.

**Pull request merged**